src/skimage/transform/_geometric.py

```diff
@@ -934,26 +934,50 @@ def estimate(self, src, dst):
         return self._estimate(src, dst) is None
 
 
+_WARP_NOTES = '''Notes
+    -----
+
+    Whether a particular (positive) rotation will appear to be clockwise or
+    anti-clockwise depends on the way you apply the transform to an image, for
+    example using :func:`skimage.transform.warp`.  If you are using ``warp``
+    from the ``skimage`` namespace, ``warp``'s default behavior is to consider
+    the first coordinate axis to refer to (floating point) indices in the
+    *second axis* of an image array, and the second coordinate axis to refer to
+    the *first axis* of the image array.  This is known as the image processing
+    or "xy" convention.  In this case, a positive rotation, once applied, will
+    be clockwise.  If you are using ``warp`` from the ``skimage2`` namespace
+    (available in Scikit-image versions >= 2), the default behavior will be to
+    consider the first coordinate axis to refer to the first axis of the image
+    array. This is the "ij", "array" or "Numpy" convention.  In that case,
+    ``warp`` will generate a rotation that is anti-clockwise.'''
+
+
 class ProjectiveTransform(_HMatrixTransform):
-    r"""Projective transformation.
+    __doc__ = rf"""Projective transformation.
 
     Apply a projective transformation (homography) on coordinates.
 
-    For each homogeneous coordinate :math:`\mathbf{x} = [x, y, 1]^T`, its
+    In what follows, ``x`` refers to coordinates from the first coordinate
```

Comment:
This is pretty confusing, given that we now have both `xy` and `ij` conventions. But, I don't see a better way.

Reply:
No, here I trying to point out that this is a general description of the transform, where $x$ in the description refers to the first axis (whatever that is) and $y$ in the description refers to the second axis (whatever that is).  In Skimage2, we can rewrite to use $i$ and $j$ if we want, but we have to leave it this way for Skimage1 ...